.gitignore

```diff
@@ -1 +1 @@
-/node_modules/
+node_modules/
```

Comment:
removing the slash from the beginning because could have node_modules inside sample folders